assets/src/block-templates/highlighted-cta/template.js

```diff
@@ -0,0 +1,47 @@
+import mainThemeUrl from '../main-theme-url';
```

Comment:
Keep an eye this block's refactor, please!
https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1205/files